### PR TITLE
Added info on block time and timeout settings for IBFT 2.0

### PR DESCRIPTION
**Pull request by MadelineMurray**
Added info on block time and timeout settings for IBFT 2.0

### CODE REVIEW

File: docs/Consensus-Protocols/IBFT.md [hunk 1]
```diff
@@ -77,6 +53,48 @@ Properties that have specific values in IBFT 2.0 genesis files are:
...
 
+### Extra Data 
+
+The `extraData` property is RLP encoded. RLP encoding is a space efficient object serialization scheme 
+used in Ethereum. Use the Pantheon subcommand [`rlp encode`](../Reference/Pantheon-CLI-Syntax.md#rlp) 
+to generate the `extraData` RLP string to include in the genesis file. 
+
+!!! example                                        
+    ```bash
+    pantheon rlp encode --from=toEncode.json --to=rlpEncodedExtraData
```
Comment (NicolasMassart) [inline docs/Consensus-Protocols/IBFT.md]: you can also omit the --to option and simply have it displayed on the console, simpler to copy from the console as anyway you copy the value from the file in the next step.

File: docs/Consensus-Protocols/IBFT.md [hunk 2]
```diff
@@ -77,6 +53,48 @@ Properties that have specific values in IBFT 2.0 genesis files are:
...
+    [
+     "9811ebc35d7b06b3fa8dc5809a1f9c52751e1deb"
+    ]
+    ``` 
+
+Copy the RLP encoded data from the `rlpEncodedExtraData` file to the `extraData` in the genesis file. 
+
+### Block Time 
+
+When a block is received, the block time and timeout timers are started. When `blockperiodseconds` is reached,
```
Comment (NicolasMassart) [inline docs/Consensus-Protocols/IBFT.md]: for me it's not clear what reaches `blockperiodseconds`. I guess it's block time timer but it's not obvious. Same for `requesttimeoutseconds` even if given the name it probably links to the timeout timer.

File: docs/Consensus-Protocols/IBFT.md [hunk 3]
```diff
@@ -77,6 +53,48 @@ Properties that have specific values in IBFT 2.0 genesis files are:
...
+round is then started, and the block time and timeout timers are reset. When `blockperiodseconds` is reached, the next 
+new block is proposed. 
+
+If `requesttimeoutseconds` is reached before the proposed block is added, a round change occurs, and the block time and 
+timeout timers are reset. The timeout period for the new round is 2 X `requesttimeoutseconds`. The timeout period continues 
+to double each time a round fails to add a block. 
+
+The time from proposing a block to the block being added is small (around 1s) even in networks
+with geographically dispersed validators. Setting `blockperiodseconds` to your desired block time and `requesttimeoutseconds`
+to 2 X `blockperiodseconds` generally results in blocks being added every `blockperiodseconds`.
```
Comment (NicolasMassart) [inline docs/Consensus-Protocols/IBFT.md]:
```suggestion
to two times the `blockperiodseconds` generally results in blocks being added every `blockperiodseconds`. 
```

File: docs/Consensus-Protocols/IBFT.md (hunk 2 above)
Comment (rain-on) [inline docs/Consensus-Protocols/IBFT.md]: Maybe "When a new chain head is received"?

Comment (rain-on) [inline docs/Consensus-Protocols/IBFT.md]:
```suggestion
When a block is received, the block time and round timeout timers are started. When `blockperiodseconds` is reached, 
```

File: docs/Consensus-Protocols/IBFT.md [hunk 4]
```diff
@@ -77,6 +53,48 @@ Properties that have specific values in IBFT 2.0 genesis files are:
...
+Copy the RLP encoded data from the `rlpEncodedExtraData` file to the `extraData` in the genesis file. 
+
+### Block Time 
+
+When a block is received, the block time and timeout timers are started. When `blockperiodseconds` is reached, 
+a new block is proposed. Generally, the proposed block is added before reaching `requesttimeoutseconds`. A new 
+round is then started, and the block time and timeout timers are reset. When `blockperiodseconds` is reached, the next 
+new block is proposed. 
+
+If `requesttimeoutseconds` is reached before the proposed block is added, a round change occurs, and the block time and
```
Comment (rain-on) [inline docs/Consensus-Protocols/IBFT.md]: This sentence need to appear before the above "..Generally, the proposed block ...."

File: docs/Consensus-Protocols/IBFT.md [hunk 5]
```diff
@@ -77,6 +53,48 @@ Properties that have specific values in IBFT 2.0 genesis files are:
...
+timeout timers are reset. The timeout period for the new round is 2 X `requesttimeoutseconds`. The timeout period continues 
+to double each time a round fails to add a block. 
+
+The time from proposing a block to the block being added is small (around 1s) even in networks
+with geographically dispersed validators. Setting `blockperiodseconds` to your desired block time and `requesttimeoutseconds`
+to 2 X `blockperiodseconds` generally results in blocks being added every `blockperiodseconds`. 
+
+!!! example 
+    An internal PegaSys IBFT 2.0 testnet has 4 geographically dispersed validators in Sweden, 
+    Sydney, and North Viringia (2). With a `blockperiodseconds`of 5 and a `requesttimeoutseconds` of 10
```
[the remark this answers is not in the document]
Comment (MadelineMurray) [reply docs/Consensus-Protocols/IBFT.md]: Fixed

File: docs/Consensus-Protocols/IBFT.md (hunk 1 above)
Comment (MadelineMurray) [reply docs/Consensus-Protocols/IBFT.md]: Fixed

File: docs/Consensus-Protocols/IBFT.md (hunk 2 above)
Comment (MadelineMurray) [reply docs/Consensus-Protocols/IBFT.md]: Fixed

File: docs/Consensus-Protocols/IBFT.md (hunk 3 above)
Comment (MadelineMurray) [reply docs/Consensus-Protocols/IBFT.md]: Fixed

File: docs/Consensus-Protocols/IBFT.md (hunk 2 above)
Comment (MadelineMurray) [reply docs/Consensus-Protocols/IBFT.md]: Fixed

**Pull request merged**